components/prize_pool/commons/prize_pool.lua

```diff
@@ -474,4 +488,17 @@ function Placement:_parseOpponentArgs(input, date)
 	return Opponent.resolve(opponentData, date)
 end
 
+function Placement:_setUsdFromRewards(prizesToUse, prizeTypes)
+	for _, opponent in ipairs(self.opponents) do
+		if not opponent.prizeRewards[PRIZE_TYPE_USD .. 1] then
+			local usdReward = 0
+			for _, prize in pairs(prizesToUse) do
+				local localMoney = opponent.prizeRewards[prize.id] or self.prizeRewards[prize.id] or 0
+				usdReward = usdReward + prizeTypes[prize.type].convertToUsd(prize.data, localMoney, opponent.date)
```

Comment:
do we maybe want to do a == 0 check here?
so that we do not call the conversion if local currency is 0 (return value will be 0 in that case too, so it would save runtime)

Reply:
Added